lang/swi-prolog-devel/Portfile

```diff
@@ -53,7 +53,8 @@ depends_lib    \
 	port:ossp-uuid        \
 	port:libarchive       \
 	port:pcre             \
-	port:db53
+	port:db53             \
+	port:gperftools
```

Comment:
Please sort all of the lib dependencies alphabetically. And do the same for the build dependencies.

Reply:
Done.